### ISSUE TITLE
[YSQL][LST] ERROR:  Invalid argument: Unknown session: 3

### ISSUE THREAD

**Issue by def-** [yugabyte/yugabyte-db#13085]
Jira Link: [DB-2799](https://yugabyte.atlassian.net/browse/DB-2799)
### Description

```
$ cd ~/code/yugabyte-db
$ git checkout 206292f0de9d1e04f42ec9fc04c832d1cb0bb34d
$ git revert 85ac8d87d99a214d99f6e701334b3162bfedc7e5.
$ ./yb_build.sh release
$ bin/yb-ctl --replication_factor 1 create --tserver_flags=yb_enable_read_committed_isolation=true --master_flags=yb_enable_read_committed_isolation=true
$ cd ~/code/yb-long-system-test
$ git checkout 88de974e627a6bd26d2e2141d36be488c5322761
$ python3.9 ./long_system_test.py --nodes=127.0.0.1:5433 --threads=10 --runtime=0 --max-columns=10 --seed=032216 --complexity=full
2022-06-28 07:02:37,368 MainThread INFO     Reproduce with: git checkout 1d35663a && ./long_system_test.py --nodes=127.0.0.1:5433 --threads=10 --runtime=0 --max-columns=10 --seed=032216 --complexity=full
2022-06-28 07:02:38,249 MainThread INFO     Database version: PostgreSQL 11.2-YB-2.15.1.0-b0 on x86_64-pc-linux-gnu, compiled by clang version 12.0.1 (https://github.com/yugabyte/llvm-project.git bdb147e675d8c87cee72cc1f87c4b82855977d94), 64-bit
2022-06-28 07:02:38,251 MainThread INFO     Creating tables for database db_lst_032216
2022-06-28 07:02:49,261 MainThread INFO     Starting worker_0: RandomSelectAction, SetConfigAction
2022-06-28 07:02:49,262 MainThread INFO     Starting worker_1: CreateIndexAction, DropIndexAction, SetConfigAction, AddColumnAction
2022-06-28 07:02:49,263 MainThread INFO     Starting worker_2: CreateIndexAction, DropIndexAction, SetConfigAction, AddColumnAction
2022-06-28 07:02:49,263 MainThread INFO     Starting worker_3: CreateIndexAction, DropIndexAction, SetConfigAction, AddColumnAction
2022-06-28 07:02:49,264 MainThread INFO     Starting worker_4: RandomSelectAction, SetConfigAction
2022-06-28 07:02:49,265 MainThread INFO     Starting worker_5: SingleInsertAction, SingleUpdateAction, SingleDeleteAction, BulkInsertAction, BulkUpdateAction, SetConfigAction
2022-06-28 07:02:49,266 MainThread INFO     Starting worker_6: RandomSelectAction, SetConfigAction
2022-06-28 07:02:49,266 MainThread INFO     Starting worker_7: CreateIndexAction, DropIndexAction, SetConfigAction, AddColumnAction
2022-06-28 07:02:49,267 MainThread INFO     Starting worker_8: SingleInsertAction, SingleUpdateAction, SingleDeleteAction, BulkInsertAction, BulkUpdateAction, SetConfigAction
2022-06-28 07:02:49,268 MainThread INFO     Starting worker_9: RandomSelectAction, SetConfigAction
2022-06-28 07:02:59,278 MainThread INFO     Worker queries/s: [009.2][000.9][001.9][000.4][010.8][009.7][008.5][000.5][009.0][010.3]
[...]
2022-06-28 11:16:51,358 MainThread INFO     Worker queries/s: [000.0][000.0][000.0][000.0][000.0][000.0][000.0][000.0][000.0][000.0]
2022-06-28 13:06:08,429 worker_0   ERROR    Unexpected query failure: InternalError_
Query: SELECT -84.44579454615973, FALSE, -17, '97', '[1937-02-13,2004-07-02]'::DATERANGE, CAST((octet_length(CAST((coalesce(upper(('(1967-10-12,2011-09-26)'::DATERANGE)), ('1991-05-07'))) AS TEXT))) AS BIGINT), numrange((CAST((-98) AS NUMERIC)), NULL), 6, -2147483648, '(1943-09-28,1964-07-13)'::DATERANGE, (('(-81,1)'::INT2RANGE) * ('(-35,-3]'::INT2RANGE)) * (range_merge((('(,88)'::INT2RANGE) * ('[-33,70]'::INT2RANGE)), (range_merge(('(-26,-4)'::INT2RANGE), (range_merge((('(16,35]'::INT2RANGE) * (('[-60,2)'::INT2RANGE) * ('(,-15)'::INT2RANGE))), ((range_merge(('[1,79)'::INT2RANGE), ('(40,)'::INT2RANGE))) * ('(,-64]'::INT2RANGE)))))))), '1913-11-17', '{"a": 5, "b": ["0", "1", "2", "3", "4", "5", "6", "7"], "c": false}'::json, -32768, -2147483648, FALSE, count(tg4_0.c16_numrange), '{"a": 6, "b": ["0", "1", "2", "3", "4"], "c": false}'::jsonb, '[8,66)'::INT2RANGE, '{"a": 4, "b": ["0", "1", "2", "3", "4", "5", "6"], "c": true}'::json, '{"a": 8, "b": ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"], "c": false}'::jsonb, -76.29417827824321, '83', CAST((32767) AS FLOAT4) FROM tg4_0 WHERE ('(-90,-51)'::INT2RANGE) != ('(-27,48)'::INT2RANGE) LIMIT 12 OFFSET 20;
  values: None
  runtime: 2022-06-28 10:57:34.797 - 2022-06-28 13:06:07.225
  supports explain: True
  supports rollback: True
  affected rows: None
Action: RandomSelectAction
Error class: InternalError_
Error code: XX000
Error message: ERROR:  Invalid argument: Unknown session: 3
Transaction isolation level: repeatable read
DB Node: host: 127.0.0.1, port: 5433
DB Backend PID: 2839
```
I haven't seen this failure before, happened on my dev server. A 6 hour run is extremely long, I was trying to reproduce other issues. logs: 
[postgresql-2022-06-28_105732.log](https://github.com/yugabyte/yugabyte-db/files/9003693/postgresql-2022-06-28_105732.log)
[yb-tserver.dev-server-dfelsing.dfelsing.log.INFO.20220628-070203.2213.tar.xz.zip](https://github.com/yugabyte/yugabyte-db/files/9003740/yb-tserver.dev-server-dfelsing.dfelsing.log.INFO.20220628-070203.2213.tar.xz.zip)

**Comment by def-** [comment]
This is still happening in current code (14fcb49214999f4fddf1e28275c7df4d92e2318b):
```
[deen@devp yugabyte-db]$ cat /home/deen/yugabyte-data/node-2/disk-1/yb-data/tserver/logs/yb-tserver.FATAL.details.2022-12-23T01_23_58.pid773744.txt
F20221223 01:23:58 ../../../../../../src/yb/yql/pggate/pggate.cc:1760] Check failed: _s.ok() Bad status: Invalid argument (yb/tserver/pg_client_service.cc:429): Unknown session: 634
    @     0x7fafdc230e6a  google::LogMessage::SendToLog()
    @     0x7fafdc2318d0  google::LogMessage::Flush()
    @     0x7fafdc2341a9  google::LogMessageFatal::~LogMessageFatal()
    @     0x7fafe06913ba  yb::pggate::PgApiImpl::ClearSeparateDdlTxnMode()
    @     0x5620e041184b  YBResetDdlState
    @     0x5620e04117b4  YBTxnDdlProcessUtility
    @     0x5620e029e15a  PortalRunUtility
    @     0x5620e029d847  PortalRunMulti
    @     0x5620e029d0f6  PortalRun
    @     0x5620e029a816  yb_exec_simple_query_impl
    @     0x5620e029adb8  yb_exec_query_wrapper_one_attempt
    @     0x5620e0297e08  PostgresMain
    @     0x5620e0206bfc  BackendRun
    @     0x5620e0206043  ServerLoop
    @     0x5620e02022db  PostmasterMain
    @     0x5620e015fe9d  PostgresServerProcessMain
    @     0x5620e0160622  main
    @     0x7fafdff76825  __libc_start_main
    @     0x5620dff097b9  _start
```

**Comment by fizaaluthra** [comment]
Closing as part of #13128  -- addressed in commit 5a6d54a6995b3d45dbba257abde296e91fa63e54
